packages/@okta/vuepress-site/docs/guides/mobile-idx-sdk-overview/main/index.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Overview of the mobile Identity Engine SDK
+---
+
+<div class="oie-embedded-sdk">
+
+Design the implementation of sign-in for your mobile app by understanding the
+representation of the flow, and with examples of the common parts.
+
+<ApiLifecycle access="ie" /><br>
+
+## Introduction
+
+Integrate the Okta sign-in flow into your app in one of two ways: redirect the request to
+Okta which handles the flow in it's own window, or handle each step in the flow by
+building the appropriate user interface elements, capturing the information, and updating
+Okta.
+
+This guide is an overview of building your own user interface. For information on the redirect model, see <StackSnippet snippet="redirectquickstart" inline />.
+
+## Sign-in flow
+
+Okta supports many ways of authenticating the identity of a user during sign-in. An Okta org administrator creates _policies_, different mixes of authentication methods (also
+called factors), and assigns them to apps, people, groups, and more. Policies also
+configure how many authentication methods are required (multifactor), and which methods
+are optional. This results in a large number of possible combinations for the different
+factors and steps to complete the sign-in flow.
+
+The Android and iOS Identity Engine SDKs represent the sign-in flow as a state machine.
+You initialize the machine with the details of your Okta Application Integration, request
+the initial step in the flow, and cycle through responding to steps until either the user
+is signed in, cancels, or an error occurs.
+
+<div class="common-image-format">
+
+![A diagram showing the sign-in flow.](/img/mobile-sdk/mobile-idx-basic-flow.png "A diagram that shows the sign-in flow.")
+
+</div>
+
+Each sign-in step may include one or more possible user actions, such as:
+- Choosing an authenticator, such as Okta Verify or WebAuthN.
```

Comment:
We shouldn't mention `WebAuthN`, it's not a supported feature in the mobile SDKs (yet)! Maybe replace it with SMS/Voice?

Reply:
I've replaced WebAuthN as an example, that way we don't incur the maintenance cost of updating when WebAuthN is supported.